docs/kb/frontend.md

```diff
@@ -5,9 +5,9 @@ order: 8
 
 # Frontend overview
 
-A Vue frontend app is created in the `vue` directory when a blockchain is scaffolded. To start the frontend app run `npm i && npm run serve` in the `vue` directory.
+A Vue frontend app is created in the `vue` directory when a blockchain is scaffolded. To start the frontend app run `npm i && npm run dev` in the `vue` directory.
 
-The frontend app is built using the `@ignt/vue` and `@ignt/vuex` packages. For details, see the [monorepo for Ignite CLI front-end development](https://github.com/ignite-hq/vue).
+The frontend app is built using the `@starport/vue` and `@starport/vuex` packages. For details, see the [monorepo for Ignite CLI front-end development](https://github.com/ignite-hq/web).
```

Comment:
@marinhoarthur @fadeev can you please confirm the NPM package names?

Reply:
I believe your change is correct. The frontend template is still using `@starport/vue` and `vuex`.